app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post, counter_cache: :comments_counter
+end
```

Comment:
- [ ] Comment model should have a method that updates the comments counter for a post. This should be similar to the example below:

```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```

- **[OPTIONAL]** It is a nice idea to use one of the Rails Active Callbacks after_save to call the update_comments_counter method every time a new like has been saved to the database, then make the update_comments_counter` method private, similar to:

```ruby
  after_save :update_comments_counter

  private

  def update_comments_counter
    author.increment!(:comments_counter)
  end
```